book/getting-started/vhost.rst

```diff
@@ -0,0 +1,99 @@
+Adding a vhost
+==============
+
+You need to configure a virtual host for your webserver in order to run Sulu.
+Here is an example for Apache and Nginx.
+
+## Apache
```

Comment:
New line after headings, and we use the notation with underlining the word with `-` as you already did on "Adding a vhost" with `=`